src/frameworks/ccpa_from_us_privacy_string.js

```diff
@@ -22,7 +22,7 @@ class CCPAFromUSPrivacyString extends FrameworkBase {
   }
 
   supportedCapabilities() {
-    return ['canUsePersonalInformationForTargeting', 'hasBeenNotifiedOfRights'];
+    return ['canUsePersonalInformationForTargeting', 'hasBeenNotifiedOfRights', 'isLSPACoveredTransaction'];
```

Comment:
Same query as for the other PR (since rescinded): is `isLSPACoveredTransaction` something library-users will need, rather than an internal detail?

Reply:
Yeah this is the question I also have. I've had to push and pull on the definition of _Capabilities_ and what is the right amount of information to expose externally. 

(I continue to imagine a case where we'd want a UI bit to show a state of opt out or just what sort of data-transactions might be possible on this site -- in those cases these details are important.)

Regardless you are right, I expect this to be heavily refined as it is used more often.